internal/resource/factory.go

```diff
@@ -71,7 +73,7 @@ func getManager(config *spec.Config) Manager {
 		return NewCudaManager()
 	}
 
-	if *config.Flags.GFD.VfioPciDriverMode {
+	if strings.EqualFold(*mode, "vfio") {
```

Comment:
Let's just use `mode == "vfio"` here and require lower case input.

Reply:
Fixed